src/ooxml/java/org/apache/poi/xssf/usermodel/XSSFVMLDrawing.java

```diff
@@ -264,7 +264,7 @@ protected CTShape newCommentShape(){
      *
      * @return the comment shape or <code>null</code>
      */
-    protected CTShape findCommentShape(int row, int col){
+    public CTShape findCommentShape(int row, int col){
```

Comment:
Does `protected` not work? `CTShape` is an xmlbean class that's an implementation detail of POI that shouldn't be relied upon.